subprojects/ear/src/main/java/org/gradle/plugins/ear/Ear.java

```diff
@@ -226,6 +233,19 @@ public void setLibDirName(@Nullable String libDirName) {
         this.libDirName = libDirName;
     }
 
+    /**
+     * Specifies if deploymentDescriptor should be generated.
+     * @return
+     */
+    @Input
+    public Property<Boolean> getGenerateDeploymentDescriptor() {
+        return generateDeploymentDescriptor;
+    }
+
+    public void setGenerateDeploymentDescriptor(boolean generateDeploymentDescriptor) {
```

Comment:
Please remove this setter. We haven't been consistent with this in the past, but these methods aren't necessary now.  We'll generate the appropriate setter at runtime.